src/tasks/get-plans-task.ts

```diff
@@ -0,0 +1,39 @@
+import { BaseTask } from "./base-task";
+import { Stripe } from "stripe";
+import { DatabaseTransactionHandler, Member } from "graasp";
+import { FastifyLoggerInstance } from "fastify";
+import { Plan } from "../interfaces/plan";
+import { CustomerExtra } from "../interfaces/customer-extra";
+
+export class GetPlansTask extends BaseTask<Plan[]> {
+  get name(): string {
+    return GetPlansTask.name;
+  }
+
+  constructor(member: Member<CustomerExtra>, stripe: Stripe) {
+    super(member, null, stripe);
+  }
+
+  async run(handler: DatabaseTransactionHandler, log: FastifyLoggerInstance): Promise<void> {
+    this.status = "RUNNING";
+
+    const plans = (await this.stripe.prices.list({ expand: ["data.product"] })).data
+      .filter((price) => (<Stripe.Product>price.product).metadata["type"] === "INDIVIDUAL_PLAN")
+      .map<Plan>((price) => {
+        return {
+          id: price.id,
+          name: (<Stripe.Product>price.product).name,
+          price: price.unit_amount / 100 ?? 0,
+          currency: price.currency,
+          interval: price.recurring.interval,
+          description: (<Stripe.Product>price.product).description,
+          level: Number((<Stripe.Product>price.product).metadata["level"]),
```

Comment:
What is going on here? In general, please leave more comments to make the code more readable.